tests/foreman/api/test_registration.py

```diff
@@ -227,3 +227,49 @@ def test_negative_global_registration_without_ak(module_target_sat):
     with pytest.raises(HTTPError) as context:
         module_target_sat.api.RegistrationCommand().create()
     assert 'Missing activation key!' in context.value.response.text
+
+
+def test_positive_capsule_without_registration_module(
```

Comment:
Also, just a thought, do you think should we cover CLI and UI test for same scenario as well to validate the error message and behaviour for Capsule field in UI?
```suggestion
def test_negative_capsule_registration_without_registration_enabled(
```

Reply:
A UI test can be added later. We can skip cli test case